### ISSUE TITLE
Wrong Normalization Setting in Dataloader

### ISSUE THREAD

**Issue by johnmaxh** [voletiv/self-attention-GAN-pytorch#6]
I've been trying to trian the model with poor results and realized that there is a major issue. The generated images are in the range [-1, 1] whereas the real images are in the range [0, 1]. This significantly hinders training. The following line of code should be changed to normalize=True.

https://github.com/voletiv/self-attention-GAN-pytorch/blob/0087e6d97a3a5752eff9797482fd43f6c91caedf/utils.py#L59

This ensures the real images are in the range [-1,1] to match the generated images.

**Comment by sudarshanregmi** [comment]
I was having exactly this issue. Very low contrast images. Now, after normalization, results are very good. Thanks!

**Comment by voletiv** [comment]
Thanks for this point!
I meant to route that argument to the "normalize" argument from get_parameters, which was True by default. I guess I missed that, thanks! Fixed it.
